grovedb/src/replication.rs

```diff
@@ -383,277 +193,145 @@ impl GroveDb {
                 "Unsupported state sync protocol version".to_string(),
             ));
         }
-        if version != state_sync_info.version {
-            return Err(Error::CorruptedData(
-                "Unsupported state sync protocol version".to_string(),
-            ));
-        }
 
-        if !state_sync_info.current_prefixes.is_empty()
-            || !state_sync_info.processed_prefixes.is_empty()
-        {
-            return Err(Error::InternalError(
-                "GroveDB has already started a snapshot syncing".to_string(),
-            ));
-        }
+        println!("    starting:{:?}...", util_path_to_string(&[]));
```

Comment:
as far as I know platform uses `tracing` crate and it might be better to align logging with the rest of the project

Reply:
`tracing` crate is included but in the rest of the code, `println` is used.

Reply:
cool